modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java

```diff
@@ -242,14 +225,9 @@ public int getCommerceShipmentsCount(long companyId)
 			return 0;
 		}
 
-		Stream<CommerceChannel> stream = commerceChannels.stream();
-
-		long[] commerceChannelGroupIds = stream.mapToLong(
-			CommerceChannel::getGroupId
-		).toArray();
-
 		return commerceShipmentLocalService.getCommerceShipmentsCount(
-			commerceChannelGroupIds);
+			TransformUtil.transformToLongArray(
```

Comment:
@julschong A lot of methods in this class calls `_commerceChannelService.search(companyId)`, transform the result list to `long[]` of groud ids, and use the array in a local service call.

However, they are inconsistent -- some methods do an `isEmpty` check on the returned list from `search`, but some don't.

Basically, we should avoid unnecessary local service calls wherever possible. Please add shortcut to these methods.

Reply:
@julschong CommerceSubscriptionEntryServiceImpl also has this issue.